kalite/ab_testing/data/conditions.py

```diff
@@ -10,10 +10,9 @@
     "nalanda_input": {
```

Comment:
I think the cleaner fix is just to delete this file.

No conditions, no problem.